apps/test/unit/templates/manageStudents/ManageStudentsAgeCellTest.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import {shallow, mount} from 'enzyme';
+import {expect} from '../../../util/configuredChai';
+import sinon from 'sinon';
+import {UnconnectedManageStudentAgeCell as ManageStudentAgeCell} from '@cdo/apps/templates/manageStudents/ManageStudentsAgeCell';
+
+const editStudent = sinon.spy();
```

Comment:
Tests look great, but I have one suggestion about their setup. A better way to organize these spies might be with a `beforeEach` that sets up the spies, targeted at the last two tests. (You'd have to add another describe around them) That way, no matter how these tests get called, or in what order, you can ensure each test gets a new spy.

```
describe 'ManageStudentAgeCell'
test1
test 1
  describe 'updates redux'
    before each spy setup
    test 3
    test 4
```

Here's an example: https://github.com/code-dot-org/code-dot-org/blob/bb14d5fb86942f69481b28bacb0f08337ee3c57b/apps/test/unit/lib/kits/maker/SwitchTest.js#L60

Reply:
Thanks for the idea and the example - updated!